From d32e0a7e0b4e585cabd57a6ae4857138d2f62b5b Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Wed, 2 Sep 2020 20:21:26 -0300
Subject: [PATCH 059/322] [FIO extras] fastboot: don't enable console mux by
 default

This is required for enabling console/uart output at uuu, but this
shouldn't be set by default as it doesn't work properly with every
target yet (e.g. imx7ulp).

Users can still set both options manually if they want such
functionality.

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 drivers/fastboot/Kconfig | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/fastboot/Kconfig b/drivers/fastboot/Kconfig
index 51f1db477f0..1af28b5f7ee 100644
--- a/drivers/fastboot/Kconfig
+++ b/drivers/fastboot/Kconfig
@@ -81,8 +81,6 @@ config FASTBOOT_UUU_SUPPORT
 	select CMD_GPT
 	select RANDOM_UUID
 	select CMD_GPT_RENAME
-	select CONSOLE_MUX
-	select SYS_STDIO_DEREGISTER
 	help
 	  The fastboot protocol includes "UCmd" and "ACmd" command.
 	  Be aware that you provide full access to any U-Boot command,
-- 
2.34.1

